app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,44 @@
+// Space out content a bit
+body {
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+// Everything but the jumbotron gets side spacing for mobile first views
+.header {
+  padding-right: 15px;
+  padding-left: 15px;
+}
+
+// Custom page header
+.header {
+  border-bottom: 1px solid #e5e5e5;
+  margin-bottom: 15px;
+}
+// Make the masthead heading the same height as the navigation
+.header h3 {
+  padding-bottom: 19px;

```

Comment:
Properties should be ordered line-height, margin-bottom, margin-top, padding-bottom